ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/InitialisationException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core;
+
+public class InitialisationException extends RuntimeException {
```

Comment:
```suggestion
public class InitializationException extends RuntimeException {
```

Reply:
This file got caught up accidently, now removed.